app/views/locations/_form.html.erb

```diff
@@ -192,11 +192,12 @@
 
   function locationCallback(event){
     if (event.errors.length > 0 && event.errors.includes("Error: tag already exists.")) {
-        notyNotification('relax', 3000, 'error', 'topCenter', 'Location Already Saved.');
+        notyNotification('relax', 3000, 'error', 'topRight', 'Location Already Saved.');
       } else if (event.saved.length > 0) {
-        notyNotification('relax', 3000, 'success', 'topCenter', 'Location Saved');
+        notyNotification('relax', 3000, 'success', 'topRight', 'Location Saved');
       } else {
-        notyNotification('relax', 3000, 'error', 'topCenter', 'Something went wrong. Contact administrator.');
+        notyNotification('relax', 3000, 'error', 'topRight', 'Something went wrong. Contact administrator.');
+      }
```

Comment:
```suggestion
```
Hi @joshuaretallick I think line 200 is the issue...you have added an extra bracket `}` and this is making the map not get loaded and thus a test failing... sorry I am catching this late this PR should be merged after you fix this...

Reply:
Oh and you haven't been a pain...we just discovered the real cause of the issue now 😅 ...debugging!

Reply:
Great catch Ruth thanks :tada: